google/cloud/firestore_v1/aggregation.py

```diff
@@ -20,6 +20,7 @@
 """
 from __future__ import annotations
 
+from datetime import datetime
```

Comment:
nit: just importing datetime as a module is probably better? This import overrides the "datetime" module name with the class name, which would get a bit weird if we have to use other datetime imports in the future

And it makes some of the type annotations a bit ambiguous